server/data/quotes.json

```diff
@@ -120,8 +125,8 @@
     "sentence": "Learning is a treasure that will follow its owner everywhere."
   },
   {
-    "author": "Chinese Proverb",
-    "sentence": "To learn a language is to have one more window from which to look at the world."
+    "author": "Robert M. Harrison",
+    "sentence": "Learning a foriegn language will not only help you, perfecting your world it will also help you perfect others."
```

Comment:
Please fix typo on the word **foreign**.

Reply:
Done